src/universal/modules/meeting/components/CheckinCard/CheckinCard.js

```diff
@@ -3,48 +3,82 @@ import look, {StyleSheet} from 'react-look';
 import theme from 'universal/styles/theme';
 import Avatar from 'universal/components/Avatar/Avatar';
 import PushButton from 'universal/components/PushButton/PushButton';
+import {cashay} from 'cashay';
+import makePushURL from 'universal/modules/meeting/helpers/makePushURL';
+import {CHECKIN} from 'universal/utils/constants';
+import {withRouter} from 'react-router';
+import withHotkey from 'react-hotkey-hoc';
 
 const combineStyles = StyleSheet.combineStyles;
 
 let styles = {};
 
-const CardButtons = () => {
+const makeCheckinHandler = (isCheckedIn, teamId, teamMemberId) => {
```

Comment:
Now that I look at it again, these functions should be built in the park & passed in. A card has no business knowing or caring about the teamId, teamMemberId, etc.